tests/unit_tests/rng/include/statistics_check_test.hpp

```diff
@@ -44,6 +44,7 @@
 
 #define POISSON_ARGS 0.5
 
+using namespace cl;
```

Comment:
Why do you need to open the `cl` namespace if you then qualify everything with `cl::` in the code below?

Reply:
I don't see any good reason to do it so. I will remove the qualifications from below, to reduce the diff.